src/karapace/forward_client.py

```diff
@@ -14,16 +16,30 @@
 LOG = logging.getLogger(__name__)
 
 
+T = TypeVar("T", bound=BaseModel)
```

Comment:
Should we reason around some better naming.

Reply:
Done.